src/docs/arch-session.dox

```diff
@@ -0,0 +1,44 @@
+/*! @arch_page arch-session Session
+
+@section arch_session_def Definition
+After a @ref arch-connection has been established between the application and WiredTiger, the
+application can start sending requests to WiredTiger using a session. A session is internally
+represented by WT_SESSION and plays an important role since almost all operations require one.
+
+A session can only be created through an existing connection with the API
+WT_CONNECTION::open_session. It is possible to create multiple sessions through the same connection.
+In fact, one connection can have multiple sessions but one session can only be associated to one
+connection.
+
+@section arch_session_ops Operations
+The different operations a session can request to WiredTiger are related to cursors handles, table
```

Comment:
Within a session, user can open various types of cursors to perform specified operations?

Reply:
I already say a session can have various cursors in the Cursor section. I took the headers from the documentation [here](https://source.wiredtiger.com/develop/struct_w_t___s_e_s_s_i_o_n.html#details). I will see if I can reword differently.

Reply:
Thanks for the suggestion @bvpvamsikrishna, see 7f22366dbbf139351ee9fcef71092098804fd14e
In the very first paragraph of this page, I wrote `the application can start sending requests to WiredTiger using a session`, is it confusing too since I say `can start sending requests` as well ?